packages/react-native-renderer/src/ReactNativeHostConfig.js

```diff
@@ -488,6 +488,10 @@ export function clearContainer(container: Container): void {
   // UIManager does not expose a "remove all" type method.
 }
 
+export function clearInstanceHandle(instance: Instance) {
+  uncacheFiberNode(instance);
```

Comment:
Curiously I've noticed Paper already does this for all deletions already. I can remove this?

Reply:
Where is that done?

Reply:
https://github.com/facebook/react/blob/3e13d70984a0ff091e71c20e1b29209bd83342d4/packages/react-native-renderer/src/ReactNativeHostConfig.js#L77-L86

Reply:
Yeah, then we won't need the above line. We'll just need it for Fabric.